tests/test_bre.py

```diff
@@ -719,7 +719,7 @@ class TestReplaceTemplate(unittest.TestCase):
     def test_replace_unicode_name_ascii_range(self):
         """Test replacing Unicode names in the ASCII range."""
 
-        pattern = re.compile(r"(some)(.*?)(pattern)(!)")
+        pattern = re.compile(r"(some)(.+?)(pattern)(!)")
```

Comment:
Actually this was not necessary. The problem was only with zero-with patterns. This pattern matches "some" followed by "pattern!" and isn't zero-with.

Reply:
Oh, I realize that.  The affected places where actually much, much smaller, but I also realized that `.*` wasn't "necessary" anywhere in the tests and `.+` made more since I had no places I was expected a zero width match.